### DISCUSSION TITLE
MongoDB server side Javascript

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#162] [Q&A] [answered]
Hi all,

Do the Mia-Platform plugins that interact with MongoDB require the [Server Side Javascript](https://www.mongodb.com/docs/manual/core/server-side-javascript/) enabled?

In particular, I am interested in:

- CRUD Service
- Real Time Updater
- Single View Creator

Thanks in advance!

**Comment by fredmaggiowski** [comment]
Hi, in regards of the CRUD Service there is no requirement for Server Side Javascript to be enabled.

As far as I know neither do the Real Time Updater nor the Single View Creator; @danibix95 should know better

**Comment by danibix95** [comment, marked as answer]
Hi, I can confirm we are not currently exploiting MongoDB Server Side Javascript functionality (the operators and functions mentioned in the documentation) in any of the interested services.

However, I haven't tried running them yet against an instance of MongoDB whose Server Side Javascript functionality was disabled.
